example/Iris.ipynb

```diff
@@ -259,10 +256,14 @@
     "# The classes as array can be get by calling model.classes_\n",
     "num_classes = model.n_classes_\n",
     "\n",
+    "# Create custom data ranges\n",
+    "in_range = pyope.ValueRange(pyope.DEFAULT_IN_RANGE_START, 2 ** 43 - 1)\n",
+    "out_range = pyope.ValueRange(pyope.DEFAULT_OUT_RANGE_START, 2 ** 63 - 1)\n",
+    "\n",
     "# Booster Parser will parse the tree\n",
     "#  (add fake metadata here as this testing only test the model correctness)\n",
     "min_max = {'min': 0, 'max': 100}\n",
-    "meta_min_max = MetaData(min_max)\n",
+    "meta_min_max = MetaData(min_max, in_range.end)\n",
```

Comment:
Is this change just to support the API of the upstream version of this dependency, or is it orthogonal cleanup?